package.json

```diff
@@ -85,7 +85,6 @@
     "expo-updates": "~0.16.4",
     "fast-text-encoding": "^1.0.6",
     "graphemer": "^1.4.0",
-    "he": "^1.2.0",
```

Comment:
@pfrazee flagging removal of `he`

Reply:
Pretty sure that dep install was a mistake LOL